docs/faqs.rst

```diff
@@ -18,22 +18,20 @@
 PyProf FAQs
 ===========
 
-#. How do I intercept the Adam optimizer in APEX ? ::
+**How do I intercept the Adam optimizer in APEX?** ::
 
 	import pyprof
 	import fused_adam_cuda
 	pyprof.nvtx.wrap(fused_adam_cuda, 'adam')
```

Comment:
This can now be removed. We already patch APEX including Adam and other modules.